examples/push/BUILD.bazel

```diff
@@ -9,8 +9,8 @@ oci_image(
 
 oci_push(
     name = "push_image",
-    default_tags = ["latest"],
     image = ":image",
+    image_tags = ["latest"],
```

Comment:
hmm, `image_tags` sounds wrong to me since it's the manifest that gets tagged within that repository. how about `repotags` so we have a unified naming with `oci_tarball`? 

https://github.com/bazel-contrib/rules_oci/blob/main/docs/tarball.md#oci_tarball

Reply:
okay, renamed to repotags